src/atom.rs

```diff
@@ -1,7 +1,9 @@
 // Chemfiles, a modern library for chemistry file reading and writing
 // Copyright (C) 2015-2018 Guillaume Fraux -- BSD licensed
-use std::marker::PhantomData;
-use std::ops::{Deref, DerefMut, Drop};
+use std::{
+    marker::PhantomData,
+    ops::{Deref, DerefMut, Drop},
+};
```

Comment:
Are these rustfmt changes? I did not have them when running locally, and I find they make the code less readable.